src/align/hts/sam/Makefile

```diff
@@ -0,0 +1,18 @@
+RAGEL=ragel
```

Comment:
Should we have a root-level (perhaps under `./src`) makefile that calls all these little makefiles to compile the ragel source to Julia source?